test/solver/TestNullSolver.cpp

```diff
@@ -0,0 +1,140 @@
+//
+// TestNullSolver
+// This programs tests the NullSolver for a Gaussian source.
+//   Usage:
+//     srun ./TestNullSolver --info 5
+//
+//   Example:
+//     srun ./TestNullSolver --info 5
+//
+
+#include "Ippl.h"
+
+#include <Kokkos_MathematicalConstants.hpp>
+#include <Kokkos_MathematicalFunctions.hpp>
+
+#include "Utility/IpplException.h"
+#include "Utility/IpplTimings.h"
+
+#include "PoissonSolvers/NullSolver.h"
+
+// type declarations
+using Mesh_t        = typename ippl::UniformCartesian<double, 3>;
+using Centering_t   = typename Mesh_t::DefaultCentering;
+using ScalarField_t = typename ippl::Field<double, 3, Mesh_t, Centering_t>;
+using VectorField_t = typename ippl::Field<ippl::Vector<double, 3>, 3, Mesh_t, Centering_t>;
+using Solver_t      = ippl::NullSolver<VectorField_t, ScalarField_t>;
+
+KOKKOS_INLINE_FUNCTION double gaussian(double x, double y, double z, double sigma = 0.05,
+                                       double mu = 0.5) {
+    double pi        = Kokkos::numbers::pi_v<double>;
+    double prefactor = (1 / Kokkos::sqrt(2 * 2 * 2 * pi * pi * pi)) * (1 / (sigma * sigma * sigma));
+    double r2        = (x - mu) * (x - mu) + (y - mu) * (y - mu) + (z - mu) * (z - mu);
+
+    return prefactor * exp(-r2 / (2 * sigma * sigma));
+}
+
+int main(int argc, char* argv[]) {
```

Comment:
I do not really see the purpose of this test. It does not check anything.

Reply:
I agree, it is a bit useless. I didn't know what other sort of test one could do apart from checking that a 0 output is giving correctly even if the input is non-zero. Do you have any suggestions? 
Or I we remove this test?

Reply:
I would remove this test or make it much simpler. It seems a bit much for testing nothing really. But then you can also just make a unit test.

Reply:
There are no unit tests for solvers. Should I create a directory for solver unit tests just for this?

Reply:
Ok. I thought we have unit tests for solvers [which would make sense]. Maybe remove this test for the moment and we add solver unit tests in a separate PR.

Reply:
Done.